contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -716,3 +716,23 @@ CREATE OR REPLACE VIEW information_schema_tsql.routines AS
 GRANT SELECT ON information_schema_tsql.routines TO PUBLIC;
 
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+/*
+* CONSTRAINT TABLE USAGE
+*/
+CREATE OR REPLACE VIEW information_schema_tsql.constraint_table_usage
+AS SELECT sys.db_name()::sys.nvarchar(128) AS table_catalog,
+    sys.schema_name(nr.oid)::sys.nvarchar(128) AS table_schema,
```

Comment:
Column names should be in Block letters.

Reply:
Use column "orig_name" of  `sys.babelfish_namespace_ext` to get the value for "TABLE_SCHEMA". We don't need to move the function "schema_name"